pyaedt/emit_core/results/revision.py

```diff
@@ -732,3 +732,22 @@ def set_emi_category_filter_enabled(self, category: EmiCategoryFilter, enabled:
             raise RuntimeError("This function is only supported in AEDT version 2024 R1 and later.")
         engine = self.emit_project._emit_api.get_engine()
         engine.set_emi_category_filter_enabled(category, enabled)
+
+    def get_license_session(self):
+        """Get a license_session object.
```

Comment:
```suggestion
        """Get a license_session object.
```Try to avoid including a code entity in the brief description: Get a license_session.
If you feel that the code entity must be used, place it in double backticks: Get a ``license_session`` object.

Reply:
Thanks @PipKat - fixed in the most recent commit.